### PR TITLE
fix: prevent backspace from navigating backwards

### LINKED ISSUE #1448: ClayMultiSelect: When you press backspace in input to focus on the tag the browser returns to the previous page in Firefox

### PR DESCRIPTION
**Pull request by julien**
fixes #1448

### CODE REVIEW

**Comment by matuzalemsteles** [comment]
Just started reviewing :)

File: packages/clay-multi-select/src/index.tsx [hunk 1]
```diff
@@ -219,6 +219,10 @@ const ClayMultiSelect = React.forwardRef<HTMLDivElement, IProps>(
 
 			const {keyCode} = event;
 
+			if (keyCode === BACKSPACE_KEY && !sourceItems.length) {
```
Comment (matuzalemsteles) [inline packages/clay-multi-select/src/index.tsx]:
```suggestion
			if (keyCode === BACKSPACE_KEY && !inputValue) {
```

@julien I think we'd better check `inputValue`, so we will ignore it only when the input is empty. That seems to work better.

**Comment by julien** [comment]
@matuzalemsteles thanks for the suggestion

**Comment by matuzalemsteles** [comment]
@julien I will merge and release this package today and we can use LPS to send this new version to the portal.

**Comment by julien** [comment]
Thanks @matuzalemsteles! /cc @4lejandrito

**Pull request merged**